locales/en-US/app.ftl

```diff
@@ -447,6 +447,7 @@ MenuButtons--metaInfo--logical-cpu =
     }
 
 MenuButtons--metaInfo--main-process-started = Main process started:
+MenuButtons--metaInfo--main-process-ended = ended:
```

Comment:
Can we have `Main process ended:` for clarity?

Reply:
This then looks weird in the UI

Reply:
Can you provide a screenshot?

Reply:
![image](https://user-images.githubusercontent.com/490655/185144677-6a9bc5c6-d038-425d-83ca-b63b2e4fa247.png)

Reply:
Waiting for devs to chime in, but this seems very fragile. 

The string above (`Main process started`) doesn't have a line break, it just happens to show "started:" on a new line because of the available space. Other locales won't necessarily have that behavior. 

For that reason alone, we shouldn't have a second line saying "ended:" (lowercase), because that happens to work with the current layout for English.

I wonder how important is to specify that this is for the main process, and we could just just "Start:" and "End:".

Reply:
I would second just having "Start:" and "End:": it conveys the actual meaning better

Reply:
But "Main process started" and "Main process ended" seems to be preferred for now.